comment-syntax.md

```diff
@@ -1,76 +1,119 @@
 ---
 
 # Comment Syntax
 
+This document describes the comment syntax supported by TiDB.
 TiDB supports three comment styles:
 
-- Use `#` to comment a line.
-- Use `--` to comment a line, and this style requires at least one whitespace after `--`.
-- Use `/* */` to comment a block or multiple lines.
+- Use `#` to comment a line:
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    SELECT 1+1;     # comments
+    ```
+
+    ```
+    +------+
+    | 1+1  |
+    +------+
+    |    2 |
+    +------+
+    1 row in set (0.00 sec)
+    ```
+
+- Use `--` to comment a line:
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    SELECT 1+1;     -- comments
+    ```
+
+    ```
+    +------+
+    | 1+1  |
+    +------+
+    |    2 |
+    +------+
+    1 row in set (0.00 sec)
+    ```
+    
+    And this style requires at least one whitespace after `--`:
+
+   {{< copyable "sql" >}}
+
+    ```sql
+    SELECT 1+1--1;
+    ```
+
+    ```
+    +--------+
+    | 1+1--1 |
+    +--------+
+    |      3 |
+    +--------+
+    1 row in set (0.01 sec)
+    ```
+
+- Use `/* */` to comment a block or multiple lines:
+
+   {{< copyable "sql" >}}
+
+    ```sql
+    SELECT 1 /* this is an in-line comment */ + 1;
+    ```
+
+    ```
+    +--------+
+    | 1  + 1 |
+    +--------+
+    |      2 |
+    +--------+
+    1 row in set (0.01 sec)
+    ```
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    SELECT 1+
+        -> /*
+    /*> this is a
+    /*> multiple-line comment
+    /*> */
+        -> 1;
+    ```
+
+    ```
+    +-------+
+    | 1+
+    1 |
+    +-------+
+    |     2 |
+    +-------+
+    1 row in set (0.00 sec)
+    ```
+
+## Comment syntax compatible with MySQL
+
+The same as MySQL, TiDB supports a variant of C comment style:
 
-Example:
-
-```sql
-mysql> SELECT 1+1;     # This comment continues to the end of line
-+------+
-| 1+1  |
-+------+
-|    2 |
-+------+
-1 row in set (0.00 sec)
-
-mysql> SELECT 1+1;     -- This comment continues to the end of line
-+------+
-| 1+1  |
-+------+
-|    2 |
-+------+
-1 row in set (0.00 sec)
-
-mysql> SELECT 1 /* this is an in-line comment */ + 1;
-+--------+
-| 1  + 1 |
-+--------+
-|      2 |
-+--------+
-1 row in set (0.01 sec)
-
-mysql> SELECT 1+
-    -> /*
-   /*> this is a
-   /*> multiple-line comment
-   /*> */
-    -> 1;
-+-------+
-| 1+
-
-1 |
-+-------+
-|     2 |
-+-------+
-1 row in set (0.00 sec)
-
-mysql> SELECT 1+1--1;
-+--------+
-| 1+1--1 |
-+--------+
-|      3 |
-+--------+
-1 row in set (0.01 sec)
+```
+/*! Specific code */
 ```
 
-Similar to MySQL, TiDB supports a variant of C comment style:
+or
 
 ```
-/*! Specific code */
+/*!50110 Specific code */
 ```
 
-In this comment style, TiDB runs the statements in the comment. The syntax is used to make these SQL statements ignored in other databases and run only in TiDB.
+In this style, TiDB runs the statements in the comment. This syntax is to make these SQLs ignored in other databases and executed in TiDB.
```

Comment:
```suggestion
In this style, TiDB runs the statements in the comment.
```
The latter sentence is deleted in https://github.com/pingcap/docs-cn/pull/3477/files#diff-fdb64224a64c779952b5ca1ae397de6cL118